README.md

```diff
@@ -539,7 +548,7 @@ func main() {
 }
 ```
 
-However, we can do even better than this, by invoking the function directly on declaration. This makes it much clearer, that the function logic is associated with the declared variable, which is not as clear in the previous example.
+However, we can do even better by invoking the function directly after it's declared. This makes it much clearer that the function logic is associated with the declared variable:
```

Comment:
```suggestion
However, we can do even better by invoking the function directly after it's declaration. This makes it much clearer that the function logic is associated with the declared variable:
```

Reply:
That works too :) We'd just need to get rid of the apostrophe from "it's" for it to be correct.

Reply:
ah, shit 😅 nice catch.